src/main/java/zeenea/connector/ConnectionConfiguration.java

```diff
@@ -85,6 +87,29 @@ public interface ConnectionConfiguration {
    */
   FilterConfiguration getFilters();
 
+  /**
+   * Gets a Map associated with the specified key.
+   *
+   * <p>Example: environment.variables { "DEFAULT_WAREHOUSE":"zeenea_wh",
+   * "DEFAULT_SCHEMA":"zeenea_db" }
+   *
+   * @param key the key for the Map value
+   * @return the Map associated with the key, empty if not values matched
```

Comment:
Ça me parait un peu plus clair comme ça, qu'en penses-tu ?
```suggestion
   * @return the Map associated with the key, empty if not found
```